src/sgame/sg_bot_ai.cpp

```diff
@@ -1100,7 +1100,19 @@ static AINodeStatus_t BotActionReachHealA( gentity_t *self )
 		return STATUS_FAILURE;
 	}
 
-	if ( !GoalInRange( self, 100 ) )
+	// retrieve creep size or booster's range to have proper distance
+	int targetType = self->botMind->goal.getTargetedEntity()->s.modelindex;
+	float dist = 0;
+	if ( targetType != BA_A_BOOSTER )
+	{
+		dist = BG_Buildable( targetType )->creepSize - 1.f; // minus one to make sure to step on creep
+	}
+	else
+	{
+		dist = REGEN_BOOSTER_RANGE;
```

Comment:
I kinda wonder why you don't add a `-1.0f` here, but LGTM otherwise

Reply:
good catch